libs/features/placeholders.js

```diff
@@ -20,6 +21,7 @@ const fetchPlaceholders = async (config, sheet) => {
       const placeholders = {};
       json.data.forEach((item) => {
         placeholders[item.key] = item.value;
+        window.mph[item.key] = item.value;
```

Comment:
why replicate the placeholders, rather than just exposing `fetchedPlaceholders` ?